aws/modules/vpn/endpoint.tf

```diff
@@ -0,0 +1,97 @@
+# Create Client VPN Endpoint
+
+resource "aws_security_group" "vpn" {
+  provider = aws.vpn
+
+  name_prefix = "client-vpn-endpoint-sg-${var.vpn_name}"
+  description = "Security group for Client VPN endpoint ${var.vpn_name}"
+  vpc_id      = var.vpc_id
+
+  ingress {
+    description = "Regulate ips allowed to access to the VPN"
+    from_port   = 443
+    to_port     = 443
+    protocol    = "udp"
+    cidr_blocks = var.vpn_allowed_cidr_blocks
+  }
+
+  egress {
+    description = "Allow the VPN to access the internal network, unrestricted"
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
Haven't looked into the details of the VPN config yet but just based on the security group would find the `cidr_block` and description a bit misleading.
`VPN to access the **internal network**` but cir_block `0.0.0.0/0`.